gc/gc-base/src/test/java/org/projectnessie/gc/base/TestJerseyRestGcInMemory.java

```diff
@@ -35,7 +36,8 @@
 @ExtendWith(DatabaseAdapterExtension.class)
 class TestJerseyRestGcInMemory extends AbstractRestGCRepoTest {
 
-  @NessieDbAdapter static DatabaseAdapter databaseAdapter;
```

Comment:
I have previously referred from `TestNessieJaxRsExtension`. So, we should handle there as well?

Reply:
No